src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -1233,13 +1275,20 @@ bool clang_cpp_convertert::get_function_body(
       // t1() : i(2){ }
       // is converted to
       // t1() { this->i = 2; }
```

Comment:
I'd suggest to also move this comment into the `else` branch of the member-init, which it documents. Might be misleading otherwise.